modules/ROOT/pages/configure-adv-tls-context.adoc

```diff
@@ -0,0 +1,111 @@
+= Import a TLS Context from Secrets Manager (Advanced)
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If your environment has specific security requirements or requires advanced configuration, you can import a TLS context from the secrets manager.
+
+== Before You Begin
+
+You must have the following permissions to enable inbound traffic to Runtime Fabric:
+
+* Grant access to secrets
+* Manage Secret Groups
+* Read secrets metadata
+* Write secrets
+
+These permissions are available from the Secrets Manager tab in Anypoint Access Management.
+
+. From Anypoint Platform, select *Access Management*.
++
+If you do not have access to *Access Management*, ask your organization admin (system administrator) to provide the "Manage Runtime Fabrics" permission to your user. If you have access, execute the following steps to add the permission:
+. Select *Users*.
+. Locate the row containing your username and then select the blue link in the *Username* column.
+. In the *Runtime Manager* tab, enable the *Manage Runtime Fabrics* permission for your Runtime Fabric environment.
++
+[NOTE]
+It can take up to five minutes for any permissions you add to propagate.
+
+== TLS Context Configuration Prerequisites
+
+You must perform the following configuration steps prior to adding a TLS context to a secrets group:
+
+*** Have a TLS certificate with the desired Common Name set to route traffic to Runtime Fabric.
+
+.... Create a Certificate Signing Request (CSR) for a Certificate Authority (CA). To streamline this step, larger organizations have a portal/internal service that provides certificate signing from a CA or might repurpose existing certificates. If neither of the previous options are available, follow the steps required by your CA to obtain your certificate, which usually includes the following tasks:
+
+..... Submit a CSR to CA signing order. All domains/subdomains for serving traffic must be included in the Subject Alternative Name (SAN).
+..... Prove domain ownership (DV certificates) or complete the necessary background checks for the certificate level.
+..... Download the certificate and CA path (signing path) from your CA.
+.... Review domains associated with the  (SAN) in your certificate. Many clients, including most browsers, only return a lock icon, which indicates a verified TLS connection, if the URL matches a domain in the SAN field of the certificate. 
+.... Be prepared to add or update the DNS routing record to send traffic to the external load balancer IP address or addresses that front the Runtime Fabric instance. This is the final step needed to cut in or cut over traffic.
```

Comment:
I think these are the lines we decided to remove until we can put together a more comprehensive doc on certificates.

Reply:
Thank you! My notes didn't catch all the conversation. I've removed this.